test/celeritas/random/distribution/NormalDistribution.test.cc

```diff
@@ -84,6 +84,19 @@ TEST(NormalDistributionTest, move)
     EXPECT_DOUBLE_EQ(4 * samples[1], sample_normal(rng));
 }
 
+TEST(NormalDistributionTest, copy)
+{
+    DiagnosticRngEngine<std::mt19937> rng;
+    NormalDistribution<double> sample{4, 0.5};
+
+    // Initialize with parameters but not spare values
+    NormalDistribution<double> sample_copy{sample};
+
+    EXPECT_DOUBLE_EQ(4.2258231678182074, sample(rng));
+    rng = {};
+    EXPECT_DOUBLE_EQ(4.2258231678182074, sample_copy(rng));
```

Comment:
To be a little more robust, perhaps:
```suggestion
    auto orig = sample(rng);
    rng = {};
    EXPECT_DOUBLE_EQ(orig, sample_copy(rng));
```